frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py

```diff
@@ -213,5 +213,11 @@ def format_number(self, number):
         """Format number."""
         if (number.startswith("+")):
             number = number[1:len(number)]
-
-        return number
+        if (number.startswith("91")):
+            pass
+        else:
+            number = f"91{number}"
+        if len(number)==12:
+            return number
+        else:
+            frappe.throw("Please enter the correct 10 digit number")
```

Comment:
This validation is specific to Indian numbers. We have people from outside India using this app so the validations shouldn't be specific to any country.